### ISSUE TITLE
Measure dynamic vs static component instantiation speed

### ISSUE THREAD

**Issue by yjbanov** [angular/angular#1345]
Dynamic component instantiation must be as fast as static component instantiation in ng2. The major use-case is large table with dynamic cell content.

This issue for implementing two new benchmarks:

* A list of cell components created statically
* A list of the same cell components but created dynamically

Benchmark parameters: iterations, number of cells.

# Static benchmark

```html
<static-cell *for="#cell of row.cells"></static-cell>
```

`static-cell` is a minimal component that drops a single `div` with no bindings.

# Dynamic benchmark

```html
<dynamic-cell *for="#cell of row.cells"></dynamic-cell>
```

`dynamic-cell` is a `@DynamicComponent` that loads `static-cell` using the `DynamicComponentLoader` API.

# Success metric

The numbers between the two benchmarks are within 5% of each other.

**Comment by matanlurey** [comment]
+1. I'll have a similar benchmark internally for forms of dynamic cells.

**Comment by yjbanov** [comment]
Fixed in #1547
